# Shafiq_PlosBiol_ATRX_microglia
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.15679297.svg)](https://doi.org/10.5281/zenodo.15679297)


# ATRX-null Microglia Multi-omics Analysis

This repository contains all scripts, pipelines, and documentation used in our study of ATRX deletion in microglia. Our analysis integrates RNA-seq, ATAC-seq, and transposable element (TE) profiling, with dedicated workflows for LTR classification and data visualization. All steps from raw data preprocessing to final figure generation are included to ensure full reproducibility.

## Organization

- **/RNAseq/**: Workflows and scripts for RNA-seq analysis, including gene and TE differential expression (DESeq2, TEtranscripts), bidirectional transcription quantification, and LTR solo/full-length classification.

- **/ATACseq/**: ATAC-seq preprocessing, peak calling, and differential analysis of peaks and TEs.


**Questions or issues?**  
Open an Issue on GitHub or email Alireza Ghahramani at aghahram@uwo.ca.
